### DISCUSSION TITLE
Disable all camera movement for users with motion sickness?

### DISCUSSION THREAD

**Discussion by xkamronx** [Sixze/ALS-Refactored#411] [Q&A]
Hey there,

I've had multiple players say the camera movement gives them motion sickness. I'm curious if anyone has added a setup to disable the camera bob/sway/etc in third person for a more static feel? I don't want this to be a hard coded change, as most players enjoy it, but an option for players who are sensitive to this type of movement would be awesome.

I'm currently digging in to try and find a solution, but if anyone has already looked in to this, I'd really appreciate any info you can provide.

**Comment by ameaninglessname** [comment]
If that is caused by the lag on camera rotation, you can provide an option to skip the logic in `UAlsCameraComponent::CalculateCameraRotation`

**Reply by xkamronx** [reply]
This appears to make it a bit better. I believe the movement lag is a larger part of why the users are experiencing this though, so looking at where to do exactly what you mentioned above but to make the camera' stickier' when moving, versus lagging behind movement slightly.

I thought I was going to be able to use bAllowLag when I first looked but didn't seem to be usable for that.

**Reply by xkamronx** [reply]
Edit - Engine issue with live coding, restarting editor after adding the new variable in worked as intended. Just need to find out where else to place the bool to adjust movement lag now.

I added the below to the ALSCameraComponent.h, but can't seem to access it to change the variable in the BP graph. I just realized I couldn't pull the Camera component in to the BP graph on the character to change it on a keypress. Any help there?

`UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Settings")
	bool bMotionReduction = false;`
